include/Annotations/ReturnAnnotation.h

```diff
@@ -5,19 +5,22 @@
 
 class ReturnAnnotation : public Annotation {
 private:
-  std::string field;
+  // equals -1 if not defined
+  int field;
+
+  // returns true iff field does not equal -1
   bool hasField;
```

Comment:
I had it there before to store the result of doing (psuedocode) `fieldName is not empty`, which I thought made more sense to do since the field was a string, but I can see now we don't need it. I'll remove `hasField`